### ISSUE TITLE
strange behavior when moving horizontally or vertically

### ISSUE THREAD

**Issue by Bes79** [flame-engine/flame_flare#4]
// My code

```dart
class TurtleComponent extends FlareActorComponent {
  final TurtleController turtleController;

  TurtleComponent(this.turtleController)
      : super(
      FlareActorAnimation(
        'assets/Turtle.flr',
        controller: turtleController,
        fit: BoxFit.none,
        animation: 'Walk',
        width: 400,
        height: 400,
      )
  );

//  @override
//  void update(double t) {
//    super.update(t);
//    x+=10*t;
//    width=600;
//  }
}

class MyGame extends BaseGame with TapDetector {
  final double squareSize = 128;
  TurtleComponent turtleComponent;
  TurtleController turtleController;

  bool loaded = false;

  MyGame() {
    turtleController = TurtleController();
    turtleComponent = TurtleComponent(turtleController);
    turtleComponent.x = 30;
    turtleComponent.y = 70;
    add(turtleComponent);
  }

  @override
  void render(Canvas canvas) {
    super.render(canvas);
  }

  @override
  void onTap() {
    turtleController.Walk();
  }
}
```

and this result:

![image](https://user-images.githubusercontent.com/29800408/90435824-472e5d00-e0d8-11ea-9690-b22b8e660090.png)

**Comment by renancaraujo** [comment]
What is the original dimensions of the flare project?

What happens when `x` and `y` are zero?

**Comment by renancaraujo** [comment]
Reproduced on example app. Fix on #5

**Comment by Bes79** [comment]
I don't know how to determine the original dimensions of flare project?
If x and y is zero all is good.

**Comment by renancaraujo** [comment]
Yeah, the default on flare editor is 1024x720, but there is no way we can see that before embedding into a game. Anyway, that's irrelevant now. 

Just awaiting reviews on #5 to merge it.

**Comment by renancaraujo** [comment]
Should be fixed on `0.1.0`
